libs/net/models/Areas/Editor/Models/AVOverview/AVOverviewSectionItemModel.cs

```diff
@@ -0,0 +1,93 @@
+using System.Text.Json;
+using TNO.API.Models;
+using TNO.Entities;
+
+namespace TNO.API.Areas.Editor.Models.AvOverview;
+
+/// <summary>
+/// AVOverviewSectionitem class, provides a model that represents an overview item.
+/// </summary>
+public class AVOverviewSectionItemModel : BaseTypeWithAuditColumnsModel<int>
+{
+    #region Properties
+    
+    /// <summary>
+    /// get/set - The section reference.
+    /// </summary>
+    public int AVOverviewSectionId { get; set; }
+
+    /// <summary>
+    /// get/set - The item type.    
+    /// </summary>
+    public AVOverviewItemType ItemType { get; set; }
+
+    /// <summary>
+    /// get/set - The item time.
+    /// </summary>
+    public string Time { get; set; } = "";
+
+    /// <summary>
+    /// get/set - The item summary.
+    /// </summary>
+    public string Summary { get; set; } = "";
+
+    /// <summary>
+    /// get/set - The item content id.
+    /// </summary>
+    public long ContentId { get; set; }
```

Comment:
Nullable

Reply:
I've updated the PR again; however, trying to figure out the failing test